### PR TITLE
refactor(service-portal): Remove delegations feature flags and delete old project

### PR DESCRIPTION
**Pull request by snaerseljan**
#  Remove delegations feature flags and delete old project

## Remove delegation feature flags and old project

- Deleted old delegation project, i.e. `libs/service-portal/settings/access-control`
- Remove feature flags from new delegation project, i.e. `libs/portals/shared-modules/delegations`
- Removed delegation feature flags from all backend services.
- Refactored `libs/api/domains/auth`, i.e. removed services-V1 and services-V2 and now there is only one service folder 

## Why

The new delegation project is being used now in production so old project can be deleted and feature flags are unnecessary.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/service-portal/src/lib/masterNavigation.ts [hunk 1]
```diff
@@ -31,20 +29,19 @@ export const rootNavigationItem: PortalNavigationItem = {
...
-    accessControlNavigation,
-    assetsNavigation,
-    icelandicNamesRegistryNavigation,
     financeNavigation,
+    icelandicNamesRegistryNavigation,
+    informationNavigation,
+    licenseNavigation,
+    personalInformationNavigation,
     vehiclesNavigation,
-    delegationsNavigation,
```
Comment (thordurhhh) [inline apps/service-portal/src/lib/masterNavigation.ts:48]: Even though I love having this alphabetically ordered, this has been ordered in the way in is supposed to appear in the service-portal menu 😄

Comment (snaerseljan) [reply apps/service-portal/src/lib/masterNavigation.ts:48]: ahh snap sorry will revert :).
I was just discovering vs code sort alphabetically shortcut

File: libs/api/domains/auth/src/lib/services/apiScope.service.ts [hunk 2]
```diff
@@ -1,42 +1,50 @@
...
 import { ScopeTreeNode } from '../models/scopeTreeNode.model'
 import { ApiScope } from '../models'
 import { ApiScopesInput } from '../dto/apiScopes.input'
+import {
+  DomainsApi,
+  DomainsControllerFindScopesDirectionEnum,
+} from '@island.is/clients/auth/delegation-api'
 
 @Injectable()
 export class ApiScopeService implements ApiScopeServiceInterface {
```
Comment (eirikurn) [inline libs/api/domains/auth/src/lib/services/apiScope.service.ts:14]: ~Can delete this.

**Review by eirikurn — APPROVED** [review]
LGTM with minor comment.

Next up is to remove the other delegation feature flags.

**Pull request merged**